product_state/__manifest__.py

```diff
@@ -7,9 +7,9 @@
     "author": "ACSONE SA/NV, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/product-attribute",
     "category": "Product",
-    "version": "17.0.1.1.0",
+    "version": "18.0.1.0.0",
     "license": "AGPL-3",
-    "depends": ["product", "sale"],
+    "depends": ["product", "sale", "stock"],
```

Comment:
Why adding this new dependency ?

Reply:
this sounds wrong

Reply:
Thanks for review, adding this new dependency because the` ir.model.access.csv` file references the `stock.group_stock_manager` group.
This access was added in 14.0 by aleuffre (https://github.com/OCA/product-attribute/pull/1124)
I have also asked him why the dependency on the stock module was not added (https://github.com/OCA/product-attribute/pull/1124#issuecomment-2507320889)

Reply:
@natuan9 I'd remove it and fix it w/ a glue module. Then they'll have to fix the module on v14 by backporting the work from v18. See my proposal on the PR on v14.

@rousseldenis WDYT?

Reply:
thanks for your proposal, and I will implement it in a fix commit.

Reply:
Should I create a new PR for adding a glue module?

Reply:
Yes, add a separate commit here to remove dependency and entry in security.

Then, add a new PR to add product_state_stock

Reply:
hmm not sure if that's a good name... we already have https://github.com/OCA/product-attribute/tree/14.0/product_stock_state :dizzy_face: 

What about `product_state_stock_security` or `product_state_stock_base`?

Reply:
- a seperate commit
- #1819
- #1818
They're ready for review